Champions/Lulu/R.cs

```diff
@@ -16,8 +16,17 @@ public class R : GameScript
         public void OnDeactivate(Champion owner) { }
         public void OnStartCasting(Champion owner, Spell spell, Unit target)
         {
-            owner.AddBuffGameScript("LuluR", "LuluR", spell, removeAfter: 7.0f, isUnique: true);
-            ApiFunctionManager.AddBuffHUDVisual("LuluR", 7.0f, 1, owner, removeAfter: 7.0f);
+            Particle p = ApiFunctionManager.AddParticleTarget(owner, "Lulu_R_cas.troy", target, 1);
+            var buff = target.AddBuffGameScript("LuluR", "LuluR", spell);
```

Comment:
Isn't the `ApplyEffects` section precisely for applying the effects, so they don't have to be applied here..?